package.json

```diff
@@ -103,17 +103,16 @@
     "@phenomnomnominal/tsquery": "6.1.3",
     "@playwright/test": "1.44.0",
     "@react-native/babel-preset": "0.73.21",
+    "@rnx-kit/eslint-plugin": "0.4.2",
     "@rollup/plugin-node-resolve": "13.3.0",
+    "@storybook/addons": "6.5.15",
```

Comment:
missing single version policy hoist that was caught by lint rule within apps/vr-test